python/featomic_torch/tests/clebsch_gordan/equivariant_power_spectrum.py

```diff
@@ -0,0 +1,40 @@
+import io
+
+import torch
+
+from featomic.torch import SphericalExpansion
+from featomic.torch.clebsch_gordan import EquivariantPowerSpectrum
+
+
+MAX_ANGULAR = 2
+SPHEX_HYPERS_SMALL = {
+    "cutoff": {
+        "radius": 2.5,
+        "smoothing": {"type": "ShiftedCosine", "width": 0.5},
+    },
+    "density": {
+        "type": "Gaussian",
+        "width": 0.2,
+    },
+    "basis": {
+        "type": "TensorProduct",
+        # use a small basis to make the tests faster
+        # FIXME: setting max_angular=1 breaks the tests
+        "max_angular": MAX_ANGULAR,
+        "radial": {"type": "Gto", "max_radial": 1},
+    },
+}
```

Comment:
```suggestion
SPHEX_HYPERS_SMALL = {
    "cutoff": {
        "radius": 5.5,
        "smoothing": {"type": "ShiftedCosine", "width": 0.5},
    },
    "density": {
        "type": "Gaussian",
        "width": 0.2,
    },
    "basis": {
        "type": "TensorProduct",
        "max_angular": 6,
        "radial": {"type": "Gto", "max_radial": 4},
    },
}
```

Since we are not running any calculation here, we can use typical hypers.